src/rtcmix/RTcmix.cpp

```diff
@@ -72,6 +72,8 @@ int				RTcmix::rtInteractive = 0;
 int				RTcmix::rtInteractive = 1; // keep the heap going for this object
 #endif
 
+int                             RTcmix::rtOSC = 0;
```

Comment:
Lets make both the member name and the setter/getter methods a bit more descriptive of their function.
That can be as simple as RTcmix::rtUseOSC
and then useOSC() and setUseOSC(bool)

Reply:
I don't see this in the changes that are visible to me.  ???